cli/cfncluster/cfncluster.py

```diff
@@ -274,7 +277,7 @@ def poll_master_server_state(stack_name, config):
         status = ('\rMasterServer: %s\n' % state.upper())
         sys.stdout.write(status)
         sys.stdout.flush()
-    except boto.exception.BotoServerError as e:
+    except ClientError as e:
         if e.message.endswith("does not exist"):
             logger.critical(e.message)
```

Comment:
there is no "message" in e.
Should be
logger.critical(e.response.get('Error').get('Message'))
?